packages/ckeditor5-dev-release-tools/lib/tasks/commitandtag.js

```diff
@@ -39,6 +41,19 @@ export default async function commitAndTag( { version, files, cwd = process.cwd(
 		return;
 	}
 
+	if ( dryRun ) {
+		try {
+			await git.add( filePathsToAdd );
+			await tools.shExec( 'yarn lint-staged', { cwd, verbosity: 'silent', async: true } );
```

Comment:
The executed task must be customizable. While it can use _our_ defaults, integrations should be able to define their tasks.

Reply:
Adressed in [7859810](https://github.com/ckeditor/ckeditor5-dev/pull/1077/commits/7859810ad6c8a1b9659aa2cc2171824b42507622).